api/handlers/reporthandler.py

```diff
@@ -460,12 +522,17 @@ def __init__(self, params):
             raise APIReportParamsException('Limit must be an integer greater than 0.')
         if limit < 1:
             raise APIReportParamsException('Limit must be an integer greater than 0.')
+        for access_type in access_types:
+            if access_type not in AccessTypeList:
+                raise APIReportParamsException('Not a valid access type')
```

Comment:
We should decide on a "safe" limit and reject requests that go over that limit. The rejection 400 could mention that an admin should run the script instead. @ryansanford what would you say the safe uwsgi log limit would be after your testing?